test/test.downscopedclient.ts

```diff
@@ -49,6 +49,10 @@ class TestAuthClient extends AuthClient {
     throw new Error('Cannot get subject token.');
   }
 
+  set expirationTime(expirationTime: number) {
```

Comment:
I think this could also be cleared or set to null?
`set expirationTime(expirationTime: number|undefined|null) {`